package.json

```diff
@@ -29,7 +29,9 @@
     "api-documenter": "api-documenter yaml --input-folder=temp"
   },
   "dependencies": {
-    "google-gax": "^2.1.0"
+    "fs.promises": "^0.1.2",
```

Comment:
You don't want that :) @bcoe I'm shocked they let this exist on `npm`.  `fs.promises` should be baked into node.js core.

Reply:
@telpirion this needs to be removed

Reply:
Done!